syntax = "proto3";

package equation_image_to_latex;

import "uint_packed_bytes.proto";

message Image {
    UintPackedBytes uid = 1;

    enum EquationType {
        DIGITAL = 0;
        HANDWRITTEN = 1;
        UNKNOWN = 2;
    }

    EquationType equationType = 2;

    string equation_name = 3;
    string author = 4;
    string parent_section = 5;
}

message ImageStack{
    repeated Image images = 1;
}
